helm/examples/metrics/elasticsearch_metrics.yaml

```diff
@@ -1,17 +1,24 @@
 #
 # Defines an elastic sink for reporting metrics to an ElasticSearch instance
 # Settings:
-#   type                                - sink type (must be elastic for ElasticSearch support)
-#   name                                - name for the sink instance
-#   settings.countMetricSuffix          - suffix for count metrics (default: count)
-#   settings.gaugeMetricSuffix          - suffix for gauge metrics (default: gauge)
-#   settings.histogramMetricSuffix      - suffix for histogram metrics (default: histogram)
-#   settings.host                       - ElasticSearch host settings
-#   settings.host.protocol               - protocol to use, http or https (default)
-#   settings.host.name                   - host name
-#   settings.host.port                   - port number (default 9200)
-#   settings.index                       - ElasticSearch index settings
-#   settings.index.name                  - index name
+#   type                                  - sink type (must be elastic for ElasticSearch support)
+#   name                                  - name for the sink instance
+#   settings.countMetricSuffix            - suffix for count metrics (default: count)
+#   settings.gaugeMetricSuffix            - suffix for gauge metrics (default: gauge)
+#   settings.histogramMetricSuffix        - suffix for histogram metrics (default: histogram)
+#   settings.host                         - ElasticSearch host settings
+#   settings.host.protocol                - protocol to use, http or https (default)
+#   settings.host.domain                  - host domain
+#   settings.host.certificateFilePath     - path to certificate file (optional)
+#   settings.host.port                    - port number (default 9200)
+#   settings.host.authentication          - authentication settings if authentication is enabled (optional)
+#   settings.host.authentication.type     - authentication type (determines remaining settings) (only 'basic' is supported)
+#   settings.host.authentication.username - username for basic authentication (if not stored in a secret)
+#   settings.host.authentication.password - password for basic authentication (if not stored in a secret)
```

Comment:
Why is this supported?  The password would be accessible in plain test.

Reply:
The code treats the password as encrypted using the same mechanism as when read from the environment.xml. If you feel the containerized version should always have the value in a secret then I can remove it. Should it be decided it can be provided directly, I updated the comment to reflect that it is encrypted. 

Please advise if it should be removed in favor of always being a secret (see comment below on always loading as a secret)